netmiko/ssh_dispatcher.py

```diff
@@ -170,7 +168,6 @@
     "ubiquiti_edgeswitch": UbiquitiEdgeSSH,
     "vyatta_vyos": VyOSSSH,
     "vyos": VyOSSSH,
-    "watchguard_fireware": WatchguardFirewareSSH,
```

Comment:
shouldn't be removing this.

Reply:
Please check my new commit, should of fixed it